package.json

```diff
@@ -0,0 +1,75 @@
+{
+  "name": "nimble-infra",
+  "version": "0.1.0",
+  "description": "Nimble Infrastructure Template generator",
+  "author": "Nimblehq",
+  "bin": {
+    "nimble-infra": "./bin/run"
+  },
+  "homepage": "https://github.com/nimblehq/infrastructure-templates",
+  "license": "MIT",
+  "main": "dist/index.js",
+  "repository": "nimblehq/infrastructure-templates",
+  "files": [
+    "/bin",
+    "/dist",
+    "/npm-shrinkwrap.json",
+    "/oclif.manifest.json"
+  ],
+  "dependencies": {
+    "@oclif/core": "^1",
+    "@oclif/plugin-help": "^5",
+    "@oclif/plugin-plugins": "^2.0.1",
+    "generate-template-files": "^3.2.0",
+    "inquirer": "^8.2.4",
+    "inquirer-test": "^2.0.1"
+  },
+  "devDependencies": {
+    "@oclif/test": "^2",
+    "@types/chai": "^4",
+    "@types/inquirer": "^8.2.1",
+    "@types/mocha": "^9.0.0",
+    "@types/node": "^16.9.4",
+    "chai": "^4",
+    "eslint": "^7.32.0",
+    "eslint-config-oclif": "^4",
+    "eslint-config-oclif-typescript": "^1.0.2",
+    "fancy-test": "^2.0.0",
+    "globby": "^11",
+    "mocha": "^9",
+    "oclif": "^3",
+    "shx": "^0.3.3",
+    "sinon": "^14.0.0",
+    "ts-node": "^10.2.1",
+    "tslib": "^2.3.1",
+    "typescript": "^4.4.3"
+  },
+  "oclif": {
+    "bin": "nimble-infra",
+    "dirname": "nimble-infra",
+    "commands": "./dist/commands",
+    "plugins": [
+      "@oclif/plugin-help",
+      "@oclif/plugin-plugins"
+    ],
+    "topicSeparator": " ",
+    "topics": {}
+  },
+  "scripts": {
+    "build": "shx rm -rf dist && tsc -b",
+    "lint": "eslint . --ext .ts --config .eslintrc",
+    "postpack": "shx rm -f oclif.manifest.json",
+    "posttest": "yarn lint",
```

Comment:
as we have linters setup, should we take advantage of that to run the linting as part of the test workflow?

Reply:
yes, it will be run after the test: https://docs.npmjs.com/cli/v8/using-npm/scripts#npm-test
Demo: https://github.com/nimblehq/infrastructure-templates/runs/6548887783?check_suite_focus=true#step:6:14